sdcm/utils/common.py

```diff
@@ -2803,3 +2803,11 @@ def __enter__(self):
     def __exit__(self, exit_type, value, _traceback):
         # remove the temporary folder as `sudo` to cover the case when the folder owner was changed during test
         self.node.remoter.sudo(f'rm -rf {self.folder_name}')
+
```

Comment:
this function is moved from `stress_thread.py` as I needed it and fits better here